packages/cadl-ranch-specs/http/type/model/inheritance/multiple-discriminator/main.tsp

```diff
@@ -0,0 +1,222 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+
+using TypeSpec.Http;
+
+@doc("Illustrates multiple level inheritance with multiple discriminators.")
+@scenarioService("/type/model/inheritance/multiple-discriminator")
+namespace Type.Model.Inheritance.MultipleDiscriminator;
```

Comment:
Would mmaybe rename this one to nested discriminator. Multiple discriminator makes me think that there is a multiple discriminator property on the same model

Reply:
renamed the whole scenario to `nested-discriminator`.